src/entities/events/ui/event-card/event-card.module.scss

```diff
@@ -133,6 +135,10 @@
     }
   }
 
+  &:hover &__header::after {
+    @include card-corner-accent($accent-color, $isHovered: true);
+  }
+
```

Comment:
i don't get that in which cases should we use `@include media-hover`?

Reply:
I asked Kristina fixed it in her PR about Event card